router/lambda.js

```diff
@@ -10,6 +10,9 @@ server.use(files(path.join(__dirname, 'build')));
 server.use(files(path.join(__dirname, 'public')));
 
 // public routes
+server.all('/api/referrals/status/*', (req, res) => {
+  nextRequestHandler(req, res);
+}); // auth is handled by the authorizer
```

Comment:
`auth is handled by the authorizer` might be a misleading comment, cause we don't set the authoriser to check this path